CRM/Membershipextrasdefaultconfig/Installationsteps/SampleContributionPage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Class to create sample contribution page for membership extras.
+ */
+
+class CRM_Membershipextrasdefaultconfig_Installationsteps_SampleContributionPage {
```

Comment:
I would rename **Installationsteps** to **Setup** and **CreateSampleContributionPageStep** to